packages/renderless/src/number-animation/index.ts

```
@@ -0,0 +1,55 @@
+    tick()
+  }
+}
+
+export const formattedValue =
+  ({ state, props }) =>
+  () => {
+    // 类型检查
+    if (typeof state.value !== 'number' && typeof state.value !== 'string') return
+    if (typeof props.precision !== 'number') return
+    const numValue = Number(state.value)
+    if (isNaN(numValue) || !isFinite(numValue)) return
```

Biome (1.9.4)

[error] isFinite is unsafe. It attempts a type coercion. Use Number.isFinite instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isFinite instead.


(lint/suspicious/noGlobalIsFinite)